content-tree.d.ts

```diff
@@ -753,5 +753,17 @@ export declare namespace ContentTree {
             children: [TableCaption, TableBody, TableFooter];
             columnSettings: TableColumnSettings[];
         }
+        interface CustomCodeComponent extends Parent {
+          type: 'custom-code-component';
+          path: string;
+          versionRange: string;
+          lastModified: string;
+          dataIdentifier: string;
```

Comment:
What is the `dataIdentifier` referring to here? The other types in content-tree use an `id` like [this](https://github.com/Financial-Times/content-tree/pull/52) one.

Reply:
Thanks @Marlon-Buckley - I have use this property inline with the storytelling team, and this [current PR ](https://github.com/Financial-Times/cp-content-pipeline/pull/823/files#diff-1dc17746dbe5cef9949fb6fa28fa8e839b27af33008869f5776102f99be1ed85R111)they have open.
I can push back and understand why they have chosen to use `dataIdentifier` vs `id`, if you feel this is necessary?

Reply:
hey @Marlon-Buckley - agreed this made more sense so have changed it to `id` 👍